src/code/containers/modal-message-container.js

```diff
@@ -2,7 +2,8 @@ import { connect } from 'react-redux';
 import ModalAlert from '../components/modal-alert';
 import * as actions from '../actions';
 import { resetGametes } from '../modules/gametes';
-Object.assign(actions, { resetGametes });
+// merge with the other actions
+actions.resetGametes = resetGametes;
```

Comment:
Ah, the reason I eliminated the use of Object.assign in this particular case was because I didn't like the idea of having this module be responsible for loading the polyfill rather than having it handled centrally. The only reason that was necessary was because this module was being loaded by the browser before the modules that loaded the polyfill. But rather than figure that out, it was easier to eliminate the use of Object.assign() in this case.